doc-content/enterprise-only/employee-rostering/er-rotation-edit-proc.adoc

```diff
@@ -1,17 +1,30 @@
 
-You can view the rotation pattern, add, move, and delete shifts.
+A seat is an employee assignment for a particular day in a specific time bucket.
+
+A stub list is a list of employees available to be assigned to a time bucket.
+
+For more information about the rotation pattern, see <<er-rotation-con>>.
 
 .Procedure
 . To view and edit the rotation pattern in the Employee Rostering application web interface, click the *Rotation* tab.
-. To view additional days of the rotation pattern, use the horizontal scroll bar at the bottom of the browser window.
-. To add a shift template, click an empty area of the table. The application adds a shift, determining the slot and time span automatically from the location of the click.
-. To edit a shift template, click the shift template. You can set the following values:
-** *From* and *To* day and time: The exact time and duration of the shift. The day is counted from the start of the rotation period.
-** *Rotation employee*: The employee assigned to the shift by default. Select *Unassigned* if you do not want to assign any employee to this shift by default; in this case, the rostering process determines the employee in the shift without a preferred value.
+. Select a spot form the *Rotation* menu.
+. To add a time bucket, click *Add New Time Bucket*. The *Creating Working Time Bucket* dialog appears.
+. Specify a start and end time, select any additional required skills, select the days for this time bucket, and click *Save*. The time bucket appears on the *Rotation* page.
+. To create an employee stub list to add employees to the rotation, click *Edit Employee Stub List*.
+. In the *Edit Employee Stub List* dialog, click *Add Employee* and select an employee from the list.
+. Add all of the employees required for this stub list and click *Save*. The employees appear above the time buckets on the *Rotation* page.
+. Drag an employee to a time bucket. The time bucket is populated with the employee icon.
 +
-To save the changes, click *Apply*.
-. To delete a shift template, hold the *Shift* key on the keyboard and click a te shift template. Alternatively, click the shift template and then click *Delete shift template*.
+NOTE: A time bucket can only have one employee assigned to it for each day. To add multiple employees to the same time bucket, copy the time bucket and change the employee name as required.
+. To save the changes, click *Apply*.
+. To provision the schedule, click *Scheduling* and select the spot that you created the rotation for.
+. Click *Provision* and specify the date range.
+. Deselect the spots that you do not want to include in this schedule.
+. Click the arrow next to the selected spot and deselect any time buckets that you do not want to use in your schedule.
+. Click *Provision Shifts*. The calendar is populated with the time buckets.
+. Click a time bucket on the calendar to edit that time bucket.
```

Comment:
For the last two points, since this is done on the scheduling page, the calendar is populated with "shifts generated from the time buckets" and "click on a generated shift in the calendar to modify it"